src/pages/workspaces/workspace/WorkspaceMenu.js

```diff
@@ -47,9 +47,9 @@ const DynamicWorkspaceMenuContent = ({ namespace, name, onClone, onShare, onDele
   })
 }
 
-const WorkspaceMenuContent = ({ canShare, isAzureWorkspace, isLocked, isOwner, onClone, onShare, onLock, onDelete, workspaceLoaded = true }) => {
+const WorkspaceMenuContent = ({ canShare, isAzureWorkspace, isLocked, isOwner, onClone, onShare, onLock, onDelete, workspaceLoaded = false }) => {
```

Comment:
This default value of false can be removed (since the default value is to be undefined, which is falsey).